assets/js/theme/global/mobile-menu-toggle.js

```diff
@@ -96,6 +100,16 @@ export class MobileMenuToggle {
         this.$header.classList.add('is-open')
         this.$scrollView.scrollIntoView(true)
 
+        window.scrollTo({
+            top: 0,
+        })
+
+        this.$searchInputs.forEach((input) => {
```

Comment:
What will happen if there are multiple search fields? Wouldn't this code cause focus to jump among them?